app/backend/src/tests/test_public.py

```diff
@@ -42,28 +42,29 @@ def test_GetPublicMapLayer(db):
         http_body = public.GetPublicUsers(empty_pb2.Empty())
         assert http_body.content_type == "application/json"
         data = json.loads(http_body.data)
-        print(data)
+        # Sort to ensure a deterministic order
```

Comment:
I recall this having originally been in a fixed/non-random order. Do you know why it wasn't deterministic now?